types/events.d.ts

```diff
@@ -0,0 +1,129 @@
+declare namespace PIXI {
+    namespace utils {
+// https://github.com/primus/eventemitter3
+        export interface EventEmitter {
```

Comment:
Is this needed? Couldn't you just have the script load the typings in `node_modules/eventemitter3/index.d.ts`?

Reply:
Pixi classes extend `PIXI.utils.EventEmitter` and that class is defined in `utils` through jsdoc. The other thing is that interface. 

I can make script that transforms EventEmitter3 typings to what we need, but I think that its over-complicating things.

Reply:
autogen based on someone else definitions looks like a leftpad for me.

Reply:
> The typings from EventEmitter have an export as namespace EventEmitter; line that needs to be removed (not a big deal tho). 

Good point, this is why I feel we should be using import rather than generating single-files, but we can do that in the future.